po/hu.po

```diff
@@ -20,12 +20,12 @@ msgstr ""
 #: data/com.github.tchx84.Flatseal.desktop.in:3
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:3 src/aboutDialog.ui:14
 msgid "Flatseal"
-msgstr ""
+msgstr "Flatseal"
```

Comment:
Ops, this one should be left out as well.

Reply:
Done. If there are any more, you are more than welcome to change them. It is your program after all.